components/infobox/wikis/rocketleague/infobox_person_player.lua

```diff
@@ -0,0 +1,263 @@
+---
+-- @Liquipedia
+-- wiki=rocketleague
+-- page=Module:Infobox/Person/Player
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Player = require('Module:Infobox/Person')
+local String = require('Module:StringUtils')
+local Class = require('Module:Class')
+local Earnings = require('Module:Earnings')
+local Namespace = require('Module:Namespace')
+local Variables = require('Module:Variables')
+local Page = require('Module:Page')
+local YearsActive = require('Module:YearsActive')
+
+local Injector = require('Module:Infobox/Widget/Injector')
+local Cell = require('Module:Infobox/Widget/Cell')
+local Title = require('Module:Infobox/Widget/Title')
+local Center = require('Module:Infobox/Widget/Center')
+
+local _BANNED = mw.loadData('Module:Banned')
+
+local _pagename = mw.title.getCurrentTitle().prefixedText
+
+local CustomPlayer = Class.new()
+
+local CustomInjector = Class.new(Injector)
+
+local _args
+
+local _START_YEAR = '2015'
```

Comment:
Just a thought, but should we create some kind of info module on each wiki, e.g. Module:Info, which you can ask for stuff like this?

Info.getName()
Info.getStartDate()
etc

Reply:
hmmm
idk how often it would be used
and what exactly we should put into it apart from the above mentioned ones

might be a good idea though

Reply:
The start year is definitely a pretty common thing

Reply:
think we should do this via a follow-up PR (and then adjusts all the modules we have on the git that use information we put into that module)

Reply:
Agree!